### ISSUE TITLE
Easy UI to sort layers

### ISSUE THREAD

**Issue by patriciogonzalezvivo** [tangrams/tangram-play#40]

**Comment by souperneon** [comment]
@patriciogonzalezvivo do you mean data layers?

**Comment by louh** [comment]
Follow up from talking with Mike - it would be nice to have some way of visualizing the data that comes in from the vector tile sources so people understand what data is available and what is getting styled. There was some observations from SOTM that people were needed a way to understand what the vector data is and how it can be actively styled on the fly, versus pre-rendered raster tiles.

Couple of ideas:
- Autocomplete dropdown layer names based on what's in the data sources
- A way to toggle map view to show a wireframe of vector data (similar to Illustrator and other vector graphics editing software)
- It's possible this can also be a standalone vector data browser app, but it would have a lot of benefit if baked into the Tangram editor.

**Comment by souperneon** [comment]
@louh oh now I remember this from our discussion. We added this coz I was showing how they do it in Mapbox Studio and really like that. They show a full list of layers and the possible styling tags for each. This is like a reference. It's super useful and educational. 
I can see it living as a page in the documentation (ref: https://www.mapbox.com/mapbox-gl-style-spec/)

**Comment by patriciogonzalezvivo** [comment]
Hi! @souperneon, speaking w Brett he want's tangram to be data agnostic, so is not possible to have a "hardcoded" list of layers in the editor. The way to do it is as @louh was saying by reading available data on the tiles. Here is a tool that @meetar made that do exactly that: https://github.com/tangrams/explorer ( try clicking and selecting ).
Unfortunately this feature selection requires the style be setup in a particular way : /
It might be a way to do it... I will check with @bcamper

**Comment by bcamper** [comment]
This requires more discussion, it's not something where we can just hack it
in

**Comment by souperneon** [comment]
@bcamper @patriciogonzalezvivo @louh agreed! For now pointing to our vector tile and tangram documentation will be most helpful.

**Comment by patriciogonzalezvivo** [comment]
@bcamper just to clarify, I'm not suggesting "hack it in"
